docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
   validator:
     image: gcr.io/hedera-eth-bridge-test/hedera-eth-bridge-validator:${TAG}
     environment:
+      VTAG: ${TAG}
```

Comment:
Can we rename this to VERSION_TAG or just VERSION

Reply:
sure thanks